LOADERS/REMOTE_FILE_SYSTEM/REMOTE_FILE/REMOTE_FILE.py

```diff
@@ -0,0 +1,103 @@
+from urllib.parse import urlparse
+
+from flojoy import flojoy, Image, DataFrame, Grayscale, TextBlob
+from typing import Literal, Optional
+import numpy as np
+from PIL import Image as PIL_Image
+import pandas as pd
+
+import requests
+from io import BytesIO
+
+
+def get_file_url(url: str):
+    parse_result = urlparse(url)
+    if not all([parse_result.scheme, parse_result.netloc]):
+        raise ValueError(f"'{url}' is not a valid URL.")
+    return url
+
+
+@flojoy(
+    deps={
```

Comment:
Adding a TODO for myself to replace all `@flojoy(deps={...})` with  `run_in_venv` in a single PR + decide what goes in the main env